src/System.Private.CoreLib/shared/System/Diagnostics/Contracts/Contracts.cs

```diff
@@ -678,16 +589,16 @@ public static bool ForAll<T>(IEnumerable<T> collection, Predicate<T> predicate)
         /// starting from <paramref name="fromInclusive"/> to <paramref name="toExclusive"/> - 1.</returns>
         /// <seealso cref="System.Collections.Generic.List&lt;T&gt;.Exists"/>
         [Pure]
-#if FEATURE_RELIABILITY_CONTRACTS
-        [ReliabilityContract(Consistency.WillNotCorruptState, Cer.MayFail)]  // Assumes predicate obeys CER rules.
-#endif
         public static bool Exists(int fromInclusive, int toExclusive, Predicate<int> predicate)
         {
             if (fromInclusive > toExclusive)
+#if CORECLR
```

Comment:
This should just use resource string like the CoreCLR version.

Reply:
Git mess up, I have fixed it locally already before submitting the PR, but maybe I forgot to push it. Thanks for the heads-up.

Reply:
There is an existing `#if CORECLR` like this too. Could you please fix that one too?